certified-connectors/CyberProof/apiDefinition.swagger.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "1.0",
+    "version": "2.0",
     "title": "CyberProof",
```

Comment:
@Avibek I can see that your connector title in the live version is "CyberProof Actions", but you have changed it to "CyberProof". Can you please let us know the reason for this?

https://docs.microsoft.com/en-us/connectors/cyberproof/

Reply:
@sriyen-msft this was already approved and marged in the [last PR ](https://github.com/microsoft/PowerPlatformConnectors/pull/1617/files#diff-07f176dd12059f5b5cb10c84c46723dae837735c3ab1ebf4ce14cf47997a2bc5L5)

Reply:
@Avibek Apologies for not asking this question while approving your last PR. Could you please let us know the reason for changing it to "Cyberproof" ?

Reply:
@sriyen-msft Because the connector doesn't provide only CyberProof actions components, it's provide also user actions, and triggers.